### ISSUE TITLE
zeromq 2017 support

### ISSUE THREAD

**Issue by perara** [Microsoft/vcpkg#760]
Current it is not possible to install zeromq using vcpkg for Visual Studi 2017

`vcpkg install zeromq:x64-windows`  yields following error:
```


Microsoft (R) Build Engine version 15.1.548.43366
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 10.03.2017 00.31.03.
     1>Project "C:\vcpkg\buildtrees\zeromq\src\zeromq-4.2.0\builds\msvc\vs2015\libzmq\libzmq.vcxproj" on node 1 (default targets).
     1>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.Cpp.Platform.targets(57,5): error MSB8020: The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build using the v140 build tools, please install v140 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\vcpkg\buildtrees\zeromq\src\zeromq-4.2.0\builds\msvc\vs2015\libzmq\libzmq.vcxproj]
     1>Done Building Project "C:\vcpkg\buildtrees\zeromq\src\zeromq-4.2.0\builds\msvc\vs2015\libzmq\libzmq.vcxproj" (default targets) -- FAILED.

Build FAILED.

       "C:\vcpkg\buildtrees\zeromq\src\zeromq-4.2.0\builds\msvc\vs2015\libzmq\libzmq.vcxproj" (default target) (1) ->
       (PlatformPrepareForBuild target) -> 
         C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\VC\VCTargets\Microsoft.Cpp.Platform.targets(57,5): error MSB8020: The build tools for v140 (Platform Toolset = 'v140') cannot be found. To build using the v140 build tools, please install v140 build tools.  Alternatively, you may upgrade to the current Visual Studio tools by selecting the Project menu or right-click the solution, and then selecting "Retarget solution". [C:\vcpkg\buildtrees\zeromq\src\zeromq-4.2.0\builds\msvc\vs2015\libzmq\libzmq.vcxproj]

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.69

```

Any way of fixing this?

**Comment by Mixaill** [comment]
As a workaround you can install **VC++ 2015.3 v140 toolset**
![image](https://cloud.githubusercontent.com/assets/704382/23794989/48ca4402-05a4-11e7-9c5d-23d30e8d057f.png)

**Comment by perara** [comment]
That may work (I just realized that 2017 was released yesterday).
Is this a issue at the library maintainers end or vcpkg?

**Comment by ras0219-msft** [comment]
This was an issue with the portfile, which @Mixaill has fixed with #763 (yay!).

**Comment by perara** [comment]
Thanks! +1 And keep up the good work!
